### PR TITLE
new test

### PR DESCRIPTION
**Pull request by sarah-svg**
test

### CODE REVIEW

**Comment by sarah-svg** [comment]
rrr

File: test/utils.test.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+// IMPORT MODULES under test here:
+import { getRandomThrow, didUserWin } from '../utils.js';
```
Comment (breeannb) [inline test/utils.test.js:2]: great job working together to get these tests going!

File: utils.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+export function getRandomThrow(){ //removed the (e) for testing
```
Comment (breeannb) [inline utils.js:1]: nice job coming together to refactor these functions

File: test/utils.test.js (hunk 1 above)
Comment (sarah-svg) [reply test/utils.test.js:2]: thank you thats awesome

File: utils.js (hunk 2 above)
Comment (sarah-svg) [reply utils.js:1]: does that count as the tests and functions I was missing on the first turn in of rock paper scissor

Comment (sarah-svg) [reply utils.js:1]: Just wanting to make sure what was needed??

Comment (breeannb) [reply utils.js:1]: I left a comment in canvas for you to review on lab03. I gave points for testing, just one more point for displaying wins, losses and draws

**Pull request merged**